mono/mini/debugger-agent.c

```diff
@@ -7755,20 +7757,29 @@ assembly_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
 			mono_error_cleanup (error);
 			t = NULL;
 		} else {
-			if (info.assembly.name)
-				NOT_IMPLEMENTED;
+			if (info.assembly.name) {
+				mono_reflection_free_type_info (&info);
+				g_free (s);
+				mono_domain_set_fast (d, TRUE);
+				buffer_add_string (buf, error_msg);
```

Comment:
nit: the error message for this case should be something like `"unexpected assembly-qualified type \"%s\" was provided"`

Reply:
Done